pkg/storage/mvcc_incremental_iterator.go

```diff
@@ -599,14 +593,11 @@ func (i *MVCCIncrementalIterator) UnsafeValue() []byte {
 	return i.iter.UnsafeValue()
 }
 
-// NextIgnoringTime returns the next key/value that would be encountered in a
-// non-incremental iteration by moving the underlying non-TBI iterator forward.
-// Intents in the time range (startTime,EndTime] are handled according to the
-// iterator policy.
-func (i *MVCCIncrementalIterator) NextIgnoringTime() {
+// ignoreTimeHelper updates the iterator's metadata and handles intents depending on the iterator's
+// intent policy.
+func (i *MVCCIncrementalIterator) ignoreTimeHelper() {
```

Comment:
nit: `ignoreTimeHelper()` is a bit vague -- consider making it imperative, e.g. `updateIgnoringTime()` or `advanceIgnoringTime()`, in line with the existing helpers here.

Reply:
Done.